studio/sentry.server.config.ts

```diff
@@ -0,0 +1,18 @@
+// This file configures the initialization of Sentry on the server.
+// The config you add here will be used whenever the server handles a request.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
+
+  // Adjust this value in production, or use tracesSampler for greater control
+  tracesSampleRate: parseFloat(process.env.SENTRY_SERVER_SAMPLE_RATE || "1"),
```

Comment:
We don't want to have 100% sample rate by default. This can be dangerous in production. Please disable tracing by default.